proxy/http/HttpConfig.cc

```diff
@@ -1120,6 +1125,20 @@ HttpConfig::startup()
   // Local Manager
   HttpEstablishStaticConfigLongLong(c.synthetic_port, "proxy.config.admin.synthetic_port");
 
+  // The responses with the following status code WILL BE cached with negative caching enabled.
+  c.codeNegCache.add(HTTP_STATUS_NO_CONTENT);
+  c.codeNegCache.add(HTTP_STATUS_USE_PROXY);
+  c.codeNegCache.add(HTTP_STATUS_FORBIDDEN);
+  c.codeNegCache.add(HTTP_STATUS_NOT_FOUND);
```

Comment:
You can make it 25 bytes if you use a [bitmap](http://concurrencykit.org/doc/ck_bitmap_init.html).